oss-internship-2020/jsonnet/tests/jsonnet_tests.cc

```diff
@@ -0,0 +1,108 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "jsonnet_tests.h"
+
+class JsonnetTest : public JsonnetTestHelper, public testing::Test {
+ protected:
+  void SetUp() override { TestSetUp(); }
+  void TearDown() override { TestTearDown(); }
+};
+
+// Basic test
+TEST_F(JsonnetTest, SetUp_TearDown) {
+  ASSERT_FALSE(if_jsonnet_vm_was_used);
+  ASSERT_FALSE(if_input_was_read);
+}
+
+// One file evaluation to one file
+TEST_F(JsonnetTest, One_file_no_dependencies) {
+  char input_file[] = "arith.jsonnet";
+  char output_file[] = "arith_output";
+  char output_to_read[] = "tests_output/arith_output";
+  char output_to_expect[] = "tests_expected_output/arith.golden";
```

Comment:
These can be of type `constexpr char` and named like constants.

Also below in other tests.